test/plugins/default.test.js

```diff
@@ -20,6 +22,32 @@ describe('default plugin', function () {
     it('returns a 200 response', () => {
       return request(server).get('/probot').expect(200)
     })
+
+    describe('get info from package.json', () => {
+      let cwd
+      beforeEach(() => {
+        cwd = process.cwd()
+      })
+
+      const pathTo = file => path.join(__dirname, '..', 'fixtures', 'plugins', file)
+
+      it('returns the correct HTML with values', async () => {
+        const expected = fs.readFileSync(pathTo('with-package.html'), 'utf8')
+        const actual = await request(server).get('/probot').expect(200)
+        expect(actual.text).toBe(expected)
```

Comment:
Might be easier to use [jest snapshots](https://facebook.github.io/jest/docs/en/snapshot-testing.html) here instead of manually saving/loading html files.

Reply:
See, I did that, and it was acting a fool. I forget what was going wrong, I'll give it another look 👍